src/schema/provenance_schema.yaml

```diff
@@ -88,6 +94,7 @@ ACTIVITIES:
       <<: *shared_properties
       creation_action:
         type: string
+        indexed: true
```

Comment:
Out index procedure doesn't handle Activity nodes. This `creation_action` from Dataset and Publication does get indexed.

Reply:
Pushed these corrections.  I assume comment above should say, "... `creation_action` ... does **_NOT_** get indexed."